### ISSUE TITLE
Add Non-Color Indicator for Links Across Get.gov Website

### ISSUE THREAD

**Issue by SamiyahKey** [cisagov/get.gov#345]
### Story

As a user who relies on non-color indicators for navigation, I need a consistent, visible way to identify links on all pages of get.gov without relying solely on color so that I can easily distinguish links from regular text and navigate the site more effectively in compliance with **WCAG Success Criterion 1.4.1 (Use of Color)**

URL: https://get.gov/
https://get.gov/domains
https://get.gov/blog
https://get.gov/about
https://get.gov/help

### Acceptance Criteria

- [ ] All navigation-type links across the Get.gov website (including but not limited to pages on /domains, /blog, /about, and /help) should include a right-facing chevron and underline on hover to signify that the text is a clickable link.

Example:

![Image](https://github.com/user-attachments/assets/b1fe887f-d4a0-4d58-89e1-22813074b573)

### Additional Context

- This issue is present on multiple pages across the site, and a consistent solution should be implemented site-wide.

- [ ]  Work with a designer to ensure compliance with WCAG 2.1 Success Criterion 1.4.1 (Use of Color), indicating this is a critical issue for accessibility.

### Prioritization Comments
**WCAG Conformance Level:** 2.1 AA
**WCAG Standard that applies:** [1.4.1 Use of Color](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color)
**Critical Failure if Not Met?:**  Yes
**Prioritization:** High/Must Do
**Reason:** Lack of a non-color indicator for links can create barriers for users with color vision deficiencies or cognitive disabilities, leading to navigation challenges and potential frustration across the site.

### Implementation notes

Consider setting the chevrons through CSS pseudo-elements or CSS background. You can avoid having to revise templates, and keep future template implementation simple.

**Comment by katypies** [comment]
@SamiyahKey  - can you help answer @h-m-f-t 's second question above or provide your thoughts on if we should change the display rather than underlining the links?

**Comment by SamiyahKey** [comment]
@h-m-f-t sorry I didn't get notified of your response! 
@katypies thanks for the ping!

I noticed that the idea of having a link icon for sharing was voted down. However, I believe that incorporating some form of icon, typically used for navigation, could be beneficial, especially if we opt not to follow the best practice of underlining links. I'm not fully aware of the reasons behind the decision to move away from this best practice.

We could implement a simple chevron icon, and then display the underline effect on hover. However, this approach would need to be integrated into the header-link design pattern across the entire platform. Here’s an example: 

![Image](https://github.com/user-attachments/assets/490d24d1-99d6-4767-883b-4b51d5841508)

**Comment by SamiyahKey** [comment]
Just noticed this pattern for a header text on the featured summary box, why was it decided to deviate from this?: ![Image](https://github.com/user-attachments/assets/befcd9fb-1371-4a87-8aa9-06897e286c28)

**Comment by h-m-f-t** [comment]
To clarify, I'm not opposed to underlining links, and don't think we made a deliberate choice to not underline. What I (think I) would like to avoid is having all _headers_, which are [now links to themselves](https://github.com/cisagov/get.gov/issues/255), being underlined.

**Comment by SamiyahKey** [comment]
Gotcha and that makes sense! Unfortunately, we have to decide on a single pattern, as having multiple link styles breaks WCAG standards by causing cognitive load on the user (_"is this a link or is that a link? they all look different."_). 

Since we are presenting headers with an underline and icon, we should stick to that or we change to the example with only the chevron but we would have to change them all.

**Comment by h-m-f-t** [comment]
Thanks, @SamiyahKey. Here's where I'll propose that the Design team revisit the first A/C and discussion in the comments of #255 and discuss how we easily share a URL to given header but without _always_ underlining the header. cc: @witha-c @Katherine-Osos 

> As a person that helps users
> I want to easily grab a link to a specific header on a given page w/o dropping to dev tools, and in a way that aligns to WCAG, but that isn't always underlined
> So I can share the link with those that need the info at that spot

I really like the approach that GitHub uses in A/C 1 of #255.

**Comment by SamiyahKey** [comment]
Slack conversation regarding final design decision: [https://trussworks.slack.com/archives/C06UJSC6G91/p1734104853066729](url)
